File: maldives_gst_setup/config/maldives_gst_setup.py
from frappe import _

def get_data():
    return [
        {
            "module_name": "maldives_gst_setup",  # Must match modules.txt
            "label": _("Maldives GST"),
            "color": "blue",
            "icon": "octicon octicon-globe",
            "type": "module",
            "description": _("Maldives GST Setup Wizard and automation")
        }
    ]

File: maldives_gst_setup/maldives_gst_setup/workspace/maldives_gst_setup.js
frappe.provide("frappe.workspaces");

frappe.workspaces["maldives_gst_setup"] = {
  label: "Maldives GST",
  icon: "octicon octicon-globe",
  items: [
    {
      label: "GST Tools",
      items: [
        {
          type: "page",
          name: "gst-setup-wizard",
          label: "GST Setup Wizard"
        }
      ]
    }
  ]
};
